### ISSUE TITLE
Question regarding some sentences in fine-tune-whisper.md

### ISSUE THREAD

**Issue by JaejinCho** [huggingface/blog#1010]
Thank you for sharing the nice tutorial to get to know how to fine-tune whisper model in huggingface!

While reading the tutorial, I was curious about the statements:

https://github.com/huggingface/blog/blame/main/fine-tune-whisper.md#L347-L349

It says "Here we are required to train a CTC tokenizer for each dataset we use. 
One of the advantages of using an encoder-decoder architecture is that 
we can directly leverage the tokenizer from the pre-trained model."

If I understand your writing well, I think that the advantage in the above sentences is coming from Whisper (since its BPE covers extensively over multiple languages) against the usual ASR models, rather than encoder-decoder against CTC.
